metals/src/main/scala/scala/meta/internal/metals/MetalsLanguageServer.scala

```diff
@@ -1302,6 +1302,28 @@ class MetalsLanguageServer(
               new IllegalArgumentException(s"Invalid arguments: $args.")
             )
         }).asJavaObject
+      case ServerCommands.ResolveDebugClass() =>
+        //TODO: polish this
+        val args = params.getArguments().asScala
+        val je = args.head.asInstanceOf[JsonElement]
+        val p = je.as[DebugClassParams].get
+        val (target, clazz) = (for {
+          //TODO: would be good to have error messages, maybe with use of Either
+          info <- buildTargets.findByDisplayName(
+            Option(p.project).getOrElse("root")
```

Comment:
Should we have any default here?

Reply:
Or maybe we can search through all build targets if project is not defined. In case there are two identical ones we can show a warning. `Running a.Main from 'A' project, but there is another main class with that name`